### PR TITLE
Implement user timeelapsed likes

### PR DESCRIPTION
**Pull request by NeuralAlchemist**
added 

- post to contain name of the user who created it.
- post to contain likes
- ORM relationship between PostLike - User - Post. (Many PostLikes to One Post, Many PostLikes to One User)

PBI-7
`As a pet owner, I want to create a post with texts, so that I can share contents about my pet`

PBI-29
`As a pet owner, I want to like a post, so that I can express my reaction to it`

### CODE REVIEW

**Comment by NeuralAlchemist** [comment]
This PR only missed the documentation part in the definition of DONE, but making the PR so that the FE can implement these changes.

File: src/main/java/se/kth/sda/skeleton/postlikes/PostLike.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.ObjectIdGenerators;
+import se.kth.sda.skeleton.posts.Post;
+import se.kth.sda.skeleton.user.User;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+
+@Entity
+public class PostLike {
```
Comment (janko101) [inline src/main/java/se/kth/sda/skeleton/postlikes/PostLike.java:17]: just with the naming...do we plan to use the same model for a comment like?

File: src/main/java/se/kth/sda/skeleton/posts/PostService.java [hunk 2]
```diff
@@ -66,10 +78,17 @@ public Post createPost(Post newPost) {
...
      */
     public Post updatePost(Long id, Post updatedPost) {
-        postRepository.findById(id).orElseThrow(ResourceNotFoundException::new);
+        Post post = postRepository.findById(id).orElseThrow(ResourceNotFoundException::new);
+        String loggedInUserEmail = authService.getLoggedInUserEmail();
+        User loggedInUser = userRepository.findByEmail(loggedInUserEmail);
+        if(!loggedInUserEmail.equals(post.getRelatedUser().getEmail())){
+            throw new ForbiddenException();
+        }
+        System.out.println("continues after exception");
```
Comment (janko101) [inline src/main/java/se/kth/sda/skeleton/posts/PostService.java]: Can you remove print statement

File: src/main/java/se/kth/sda/skeleton/postlikes/PostLike.java (hunk 1 above)
Comment (NeuralAlchemist) [reply src/main/java/se/kth/sda/skeleton/postlikes/PostLike.java:17]: yes!

**Review by janko101 — APPROVED** [review]

**Review by eeels22 — APPROVED** [review]
Looks great, thaks

**Pull request merged**